--- a/tools/build/src/tools/darwin.jam	2020-10-29 23:37:52.000000000 -0300
+++ b/tools/build/src/tools/darwin.jam	2020-10-29 23:37:43.000000000 -0300
@@ -192,7 +192,7 @@
     #   archives in darwin is complicated.
     local archiver =
         [ common.get-invocation-command darwin
-            : libtool : [ feature.get-values <archiver> : $(options) ] : $(bin) : search-path ] ;
+            : /usr/bin/libtool : [ feature.get-values <archiver> : $(options) ] : $(bin) : search-path ] ;
     flags darwin.archive .LIBTOOL $(condition) : $(archiver[1]) ;
     if $(.debug-configuration)
     {
